src/Discovery/Discovery.css

```diff
@@ -92,103 +101,203 @@
 .discovery-header__tags-header {
   padding-left: 50px;
   cursor: default;
-  border-bottom: solid 1px #e7e7e7;
   font-weight: normal;
   font-size: 1.125rem;
+  text-transform: uppercase;
   color: #555;
 }
 
 .discovery-header__tags {
   width: 100%;
   display: flex;
   flex-direction: row;
-  justify-content: space-around;
+  justify-content: space-between;
 }
 
 .discovery-header__tag-group {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   justify-content: flex-start;
-  font-weight: bold;
-  overflow-y: scroll;
-  overflow-x: visible;
-  max-width: 290px;
-  height: 111px;
 }
 
-.discovery-header__tag-group::-webkit-scrollbar {
-  display: none;
+.discovery-header__tag-column {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  max-height: 200px;
+  overflow-y: hidden;
+  overflow-x: hidden;
 }
 
-.discovery-header__tag-btn {
-  cursor: pointer;
+.discovery-tag-column {
+  padding-right: 15px;
 }
 
-.discovery-header__tag-btn:focus, .discovery-header__tag-btn:focus-within, .discovery-header__tag-btn:focus-visible {
-  box-shadow: 0 0 5px rgba(81, 203, 238, 1);
+.discovery-tag-column--expanded {
+  overflow-y: auto;
+}
+
+.discovery-header__tag-btn {
+  cursor: pointer;
 }
 
 .discovery-tag {
+  line-height: 18px;
+  font-size: 10px;
+  box-sizing: border-box;
   border-radius: 5px;
   border-width: 2px;
-  box-sizing: border-box;
-  line-height: 18px;
-  color: #555;
   margin-bottom: 3px;
-  user-select: none;
+}
+
+.discovery-header__tag-btn:focus, .discovery-header__tag-btn:focus-within, .discovery-header__tag-btn:focus-visible {
+  box-shadow: 0 0 5px rgba(81, 203, 238, 1);
 }
 
 .discovery-tag--selected {
   color: #fff;
 }
 
-.discovery-table-container {
-  border-radius: 5px;
+.discovery-export__selected-ct {
+  color: var(--g3-color__gray);
+}
+
+.discovery-studies-container {
+  border-radius: 8px;
   background-color: #fff;
   padding: 20px;
-  overflow-x: auto;
+  overflow-x: hidden;
 }
 
-.discovery-table__header {
+.discovery-studies__header {
   width: 100%;
   margin-bottom: 12px;
   display: flex;
   align-items: baseline;
   justify-content: space-between;
 }
 
+.discovery-filters {
+  position: absolute;
+  width: 280px;
+  margin-right: 20px;
+  overflow-y: auto;
+}
+
+.discovery-search-container {
+  margin-bottom: 10px;
+}
+
 .discovery-search {
-  border-radius: 5px;
+  border-color: rgb(139, 51, 105);
```

Comment:
all these purple colors hard-coded in this PR (both in this CSS and in some other JS codes) should be removed since they are HEAL specific color schemes

HEAL should overwrite them in gitops.css

Reply:
➕ I'm working on that now 🧑‍💻